src/services/BudgetService.ts

```diff
@@ -130,4 +150,40 @@ export class BudgetService {
       )
     }
   }
+
+  static getProposalsBudgetingMinAndMaxDates(proposals: Pick<ProposalAttributes, 'start_at'>[]): {
+    minDate?: Date
+    maxDate?: Date
+  } {
+    const sorted = proposals
+      .filter((p) => p.start_at && p.start_at >= BUDGETING_START_DATE)
+      .sort((p1, p2) => p1.start_at.getTime() - p2.start_at.getTime())
+    return { minDate: sorted[0]?.start_at, maxDate: sorted[sorted.length - 1]?.start_at }
+  }
+
+  static async getBudgets(proposals: Pick<ProposalAttributes, 'start_at'>[]): Promise<CurrentBudget[]> {
```

Comment:
This will only return two budgets right? What if we return an object like `{ current: newestBudget, previous: previousBudget }`?

(Renaming oldestBudget to previousBudget)

Reply:
This name makes me think the function returns ALL budgets. Maybe `getLatestBudgets`?